### ISSUE TITLE
EJ Index Question

### ISSUE THREAD

**Issue by eipresearch** [gisumd/EJScreen#1]
Is there a file that contains the final EJ Index value? Thanks!

**Comment by gisumd** [comment]
> Is there a file that contains the final EJ Index value? Thanks!

Yes, the MD_EJScreen.zip file contains the shapefile that has the EJ Index Score. You can also view the attribute table at https://p1.cgis.umd.edu/EJScreen/

**Comment by eipresearch** [comment]
Thanks! I'm not seeing it... what is the name of the shapefile? 
![image](https://user-images.githubusercontent.com/18537812/126218733-14ae9424-fbf9-432f-a7b3-b39e25753f70.png)

Also, the way the EJ Index is calculated in the web map layer seems to be inconsistent with the methods outlined in a) Driver et al. 2019 and b) the application's help documentation. The map layer seems to be based on an unweighted average of the pollution burden and demographic characteristic percentiles, rather than a product of weighted averages... Could you clarify?
